google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java

```diff
@@ -123,8 +131,22 @@ public HttpHeaders getHeaders() {
    *
    * @since 1.14
    */
-  public final String getContent() {
-    return content;
+  public final String getContent() throws IOException {
+    if (content == null) {
+      return null;
+    }
+    ByteArrayOutputStream out = new ByteArrayOutputStream();
```

Comment:
Does this work `return new String(content, charset.name())`?

Reply:
Yes, is is also work